sematic/resolvers/cloud_resolver.py

```diff
@@ -245,7 +245,34 @@ def _detach_resolution(self, future: AbstractFuture) -> str:
         return run.id
 
     def _schedule_future(self, future: AbstractFuture) -> None:
-        run = api_client.schedule_run(future.id)
+        run = api_client.get_run(future.id)
```

Comment:
Can you use `self._get_run`? It refreshes the local cache.

Reply:
done